### PR TITLE
Ensure pool.d/ directory for php5-fpm exists.

### PR DESCRIPTION
**Pull request by htgoebel**
I just happened to delete this directory by accident.

### CODE REVIEW

**Comment by drybjed** [comment]
The `/etc/php5/fpm/pool.d/` directory is part of the `php5-fpm` package Conffiles, so it will be created at the install time:
```
Package: php5-fpm
...
Conffiles:
 /etc/init.d/php5-fpm 20f57e0d48d21f11ab751a2869c8586f
 /etc/init/php5-fpm.conf f0061301091846a1dbbc830226f44d90
 /etc/logrotate.d/php5-fpm c9d721ed5e99b85e1bcac1d5bc3b1728
 /etc/php5/fpm/php-fpm.conf 69e2a37717fc08d05b75fc24e40f0ea5
 /etc/php5/fpm/pool.d/www.conf e1940d21c5edc3a3e869ea87bbd556e6
```
I don't think that we need to ensure its creation in Ansible, unlike `/etc/php5/fpm/pool-available.d/` which isn't part of the `php5-fpm` package. But I guess it doesn't hurt to have it here.

File: tasks/main.yml [hunk 1]
```diff
@@ -70,6 +70,15 @@
...
     mode: '0644'
 
+- name: Ensure pool.d/ directory for php5-fpm exists
+  # Just for the case it has been deleted manually
+  file:
+    path: '/etc/php5/fpm/pool.d'
+    state: 'directory'
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
```
Comment (drybjed) [inline tasks/main.yml]: The mode needs to be set as `0755` since it's a directory.

Comment (htgoebel) [inline tasks/main.yml]: Right. I copied it from just above. Fix both now.

Comment (drybjed) [inline tasks/main.yml]: Thanks, looks good.

**Pull request merged**